src/screens/TestCases/Modal/CustomFilterModal.js

```diff
@@ -110,44 +217,90 @@ const CustomFilterModal = ({
                         <select
                           className="form-select"
                           aria-label="Default select example"
+                          onChange={(e) => {
+                            setFilterType(e.target.value);
+                          }}
                         >
                           <option defaultValue>Open this select menu</option>
-                          <option value="1">Equals</option>
-                          <option value="2">Does Not Equals</option>
-                          <option value="3">Begins With</option>
+                          {filterColumn === 'id' ? (
+                            <>
+                              <option value="equals">Equals</option>
+                              <option value="not equal to">Not Equal To</option>
+                              <option value="greater than">
+                                {' '}
+                                Greater Than
+                              </option>
+                              <option value="greater than or equal to">
+                                Greater Than Or Equal To
+                              </option>
+                              <option value="less than"> Less Than</option>
+                              <option value="less than or equal to">
+                                Less Than Or Equal To
+                              </option>
+                              <option value="is between">Between</option>
+                            </>
+                          ) : (
+                            <>
+                              <option value="equals">Equals</option>
+                              <option value="does not equal">
+                                Does Not Equals
+                              </option>
+                              <option value="begins with">Begins With</option>
+                              <option value="begins with">End with</option>
+                              <option value="begins with"> Contains</option>
+                              <option value="begins with">
+                                Does Not Contain
+                              </option>
+                            </>
+                          )}
                         </select>
                       </div>
+                      {/* {filterType != 'is between' && ( */}
                       <div className="col">
                         <input
-                          type="text"
+                          type={filterColumn === 'id' ? 'number' : 'text'}
                           className="form-control"
                           placeholder="Type"
+                          onChange={(e) => setFilterText(e.target.value)}
                         />
                       </div>
+                      {/* )} */}
+                      {/* {filterType === 'is between' && (
```

Comment:
remove all commented code, if not req. else give a proper comment

Reply:
resolved